optimum/intel/openvino/training_args.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass, field
+
+from transformers import TrainingArguments
+
+
+@dataclass
+class OVTrainingArguments(TrainingArguments):
```

Comment:
@vuiseng9  can you explain why some parameters are part of the configuration json/dict, and others part of the OVTrainingArguments? It is not immediately obvious to me why for example warmup_start_epoch is an item in the config, but distillation_weight and temperature are OVTrainingArguments.

Reply:
@helena-intel The json configuration specifies hyperparameters for NNCF algorithms. NNCF native distillation is a self distillation which can be limiting. OVTrainer is extended with distillation that allows larger models as teacher. OVTrainingArguments is provided as a companion to OVTrainer to standardize adaptation of existing script/task for distillation and nncf-supported algorithms. For hyperparameters of NNCF algorithms, it would be challenging to expose through OVTrainingArguments as users can mix and match NNCF algorithms.

Reply:
I'm more wondering the other way around: it is not obvious to me that distillation_weight and temperate are not "hyperparameters for NNCF algorithms", so I'm wondering why they aren't in the config but have to be specified separately in the trainer.